t/006-default_indexed-var.t

```diff
@@ -0,0 +1,413 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+# modified from https://github.com/openresty/lua-nginx-module/blob/master/t/045-ngx-var.t
+# with index always turned on
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 8) + 10;
+
+#no_diff();
+#no_long_string();
+#master_on();
+#workers(2);
+run_tests();
+
+__DATA__
+
+=== TEST 1: sanity: directive works well
+--- http_config
+    lua_kong_load_var_index default;
+
+--- config
+    location = /test {
+        content_by_lua '
```

Comment:
Let's change all to use `content_by_lua_block` in the tests. We should not be using `content_by_lua` for new test files as it is no longer recommended to use by OpenResty.

Reply:
done